Plan/src/main/java/com/djrapitops/plan/system/cache/VelocityDataCache.java

```diff
@@ -0,0 +1,28 @@
+package com.djrapitops.plan.system.cache;
+
+import com.djrapitops.plan.system.PlanSystem;
+
+import java.util.UUID;
+
+/**
+ * Velocity specific DataCache.
+ *
+ * Based on BungeeDataCache
+ *
+ * <p>
+ * Used for overriding {@link SessionCache#endSession(UUID, long)}.
+ *
+ * @author MicleBrick
+ */
+public class VelocityDataCache extends DataCache {
```

Comment:
Duplication of this class is unnecessary, as only benefit is having it in velocity specific class.
This can be sorted later by refactoring common classes into network specific names